docs/modules/mcp.md

```
@@ -0,0 +1,195 @@
+|------|-------|
+| `STEP` | Progression d'une étape principale |
+| `SOURCE` | Requête vers une source de données |
+| `AGENT` | Activité d'un agent de débat |
+| `BROWSER` | Navigation navigateur |
+| `GEOCODE` | Opération de géocodage |
+
+Le `StepContext` (context manager `async`) permet de suivre une étape avec compteur (`update()`, `increment()`).
+
+## Fichiers clés
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)